modules/preview-react/InformationHighlight/stories/InformationHighlight.mdx

```diff
@@ -0,0 +1,111 @@
+import {
+  ExampleCodeBlock,
+  SymbolDoc,
+  Specifications,
+} from '@workday/canvas-kit-docs';
+
+import * as InformationHighlight from './InformationHighlight.stories.ts';
+import { Basic } from './examples/Basic';
+import {Body} from './examples/Body';
+import {Heading} from './examples/Heading';
+import {Informational} from './examples/Informational';
+import {Caution} from './examples/Caution';
+import {Critical} from './examples/Critical';
+import {IconCritical} from './examples/CustomIconCritical';
+import {RTL} from './examples/RTL';
+
+<Meta of={InformationHighlight} />
+
+# Canvas Kit Information Highlight
+
+A container to call out important information on a page or a section of a page that the user should be aware of.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+`InformationHighlight` should be used to call out important information to the user and are paired with a meaningful
+ icon and color to differentiate information type. It should not be removed from the interface until some system or admin level action has been taken to do so.
+
+You can use an `InformationHighlight` to:
+- Prevent unwanted consequences from destructive actions
+- Reassure users in their next steps
+- Provide detail into information on the page
+- Show changes in page content
+
+### When to Consider Something Else
+- Consider a [Banner](https://workday.github.io/canvas-kit/?path=/docs/components-indicators-banner--docs) if the message is about system-wide errors or alerts, or if the message appears in response to user action.
+- Consider a [Dialog](https://workday.github.io/canvas-kit/?path=/docs/components-popups-dialog--docs) if the message is critical and requires user action or decision before proceeding
+- Consider a [Toast](https://workday.github.io/canvas-kit/?path=/docs/components-popups-toast--docs) if the message is timely, such as communicating updates on the process of an application.
+### Basic Example
+<ExampleCodeBlock code={Basic} />
+
+
+## Variants
+`InformationHighlight` has three variants and each `variant` has two types of `emphasis`.
+
+
+Variants: `Informational`, `Caution`, `Critical`
```

Comment:
```suggestion
Variants: `informational`, `caution`, `critical`
```

Reply:
this matches the actual string consumers would use in the component and what's in the model